.github/workflows/release-drafter.yaml

```diff
@@ -6,8 +6,14 @@ on:
     branches:
       - main
 
+permissions:
+  contents: read
+
 jobs:
   update_release_draft:
+    permissions:
+      contents: write  # for release-drafter/release-drafter to create a github release
+      pull-requests: write  # for release-drafter/release-drafter to add label to PR
```

Comment:
We don't use the auto-labeler functionality, so a write permission shouldn't be required.